hack/chains/release-pipeline-with-ec-demo.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+#
+# A helper script to demonstrate a pipeline that performs a release if, and only if, the
+# Enterprise Contract is met.
+#
+# Pre-requisites:
+#
+#   1. Create a Secret named "release-demo" with access to push to the dst-image-ref, and
+#      link it to the pipeline ServiceAccount (so skopeo can copy the image there):
+#      oc create secret docker-registry release-demo quay.io --docker-username=$USER --docker-password=top-secret
+#      oc secrets link pipeline release-demo --for=pull,mount
+#
+#   2. Create a Secret named "cosign-public-key" with the "cosign.pub" attribute to house the
+#      public key to be used for verifying the signature of the image and its attestation.
```

Comment:
Would be nice to add that helper command from the build-definitions repo here too. I gotta go look for it now.

Reply:
Good idea, done!